### ISSUE TITLE
MKS TFT35 Laser - mks_config.txt PROBLEM

### ISSUE THREAD

**Issue by Francesco-Rugg** [makerbase-mks/MKS-TFT#506]
After installing _MKS TFT35 Laser_ and makes change on _mks_config.txt_ (rotate the display of 180 with the formula 180:EE)
once I turn of the tft and power on again I loose all the changes that I made in the .txt file

**Comment by mks-viva** [comment]
Hi! Can you provide your config to me and i will test it.

**Comment by Francesco-Rugg** [comment]
[mks_config.txt](https://github.com/makerbase-mks/MKS-TFT/files/5892933/mks_config.txt)

**Comment by mks-viva** [comment]
It seems that everything is normal. After you update, has the name of mks_config.txt changed to ***.cur?

**Comment by Francesco-Rugg** [comment]
> It seems that everything is normal. After you update, has the name of mks_config.txt changed to ***.cur?

yes

**Comment by mks-viva** [comment]
If has change name to ***.cur, the config has loaded.

**Comment by Francesco-Rugg** [comment]
> If has change name to ***.cur, the config has loaded.

Ok but the display when I turn on again the TFT screen turn to 0 position, not 180°


One more question.
In the config file where I can set the X-Y maximum travel?

**Comment by mks-viva** [comment]
@Francesco-Rugg 
X-Y maximum travel set, you need you “$130” and "$131" command, you can refer to here:https://github.com/gnea/grbl/wiki/Grbl-v1.1-Configuration
![image](https://user-images.githubusercontent.com/16629551/107104992-1e08b100-685f-11eb-99df-8479c6ee7c9e.png)

**Comment by Francesco-Rugg** [comment]
> @Francesco-Rugg
> X-Y maximum travel set, you need you “$130” and "$131" command, you can refer to here:https://github.com/gnea/grbl/wiki/Grbl-v1.1-Configuration
> ![image](https://user-images.githubusercontent.com/16629551/107104992-1e08b100-685f-11eb-99df-8479c6ee7c9e.png)

I need to set up the TFT, not the DLC 2.0.

**Comment by Francesco-Rugg** [comment]
> Ok but the display when I turn on again the TFT screen turn to 0 position, not 180°

This problem is not resolved.

**Comment by mks-viva** [comment]
Provide you config file and firmware to me and i will test it.

**Comment by Francesco-Rugg** [comment]
> Provide you config file and firmware to me and i will test it.

@mks-viva 
8 days ago you ask me for the CONFIG file and told me that the file is OK, but still have problem

this is the file

> [mks_config.txt](https://github.com/makerbase-mks/MKS-TFT/files/5892933/mks_config.txt)

**Comment by m-sja** [comment]
I can confirm the same behavior on my unit with this firmware (English Folder):
https://github.com/makerbase-mks/MKS-TFT/blob/master/MKS-TFT-laser/MKS_TFT35_laser_V1.0.3%20Release%20file.rar
Only change in mks_config.txt is the Line for screen rotation to:
`>cfg_screen_overturn_180:0xEE`

After inserting the sd-card and first powering the unit, the screen is rotated by 180° as intended.
(The green Bootloader-Message "Booting ..." is rotated and all the Interface is also)
The File on sd-card changes to ***.cur

After first reset or Power-Cycle, the green bootloader-message "Booting ..." is still rotated, but the whole interface is changed back to default orientation. This is not changed on further reboots.

**Comment by mks-viva** [comment]
@m-sja @Francesco-Rugg 
Hi! I has test it, this is a bug, i will feedback to our engineer

**Comment by m-sja** [comment]
Hi @mks-viva 
is there any timeline when we could expect the fix?
It still looks wierd if the Interface is upside down.
![16162288373960](https://user-images.githubusercontent.com/28825834/111863908-7783f580-895e-11eb-88de-c3aff52d7808.jpg)

Best regards

**Comment by danielminson** [comment]
I also am having exactly the same problem. Please fix it ASAP
![IMG_0447](https://user-images.githubusercontent.com/49560569/112755355-aa5d6780-8fb6-11eb-9183-b604d3209537.jpg)

**Comment by danielminson** [comment]
> Hi @mks-viva
> is there any timeline when we could expect the fix?
> It still looks wierd if the Interface is upside down.
> ![16162288373960](https://user-images.githubusercontent.com/28825834/111863908-7783f580-895e-11eb-88de-c3aff52d7808.jpg)
> 
> Best regards

I am really need this issue to be fixed? Is there any update on it?

**Comment by Francesco-Rugg** [comment]
@danielminson I haven't tried yet, the problem is the same of 3 months ago?

**Comment by danielminson** [comment]
Yes, the latest version of the firmware TFT35laser is 1.0.3 and it has exactly the same issue described here. It´s such a bummer. Is there any way to tag Makerbase to actually fix the issue?

**Comment by danielminson** [comment]
@makerbase-mks @Francesco-Rugg still waiting for a fix.

**Comment by m-sja** [comment]
> @m-sja @Francesco-Rugg Hi! I has test it, this is a bug, i will feedback to our engineer

@mks-viva any update on this.
If you release the sourcecode, the community will fix all this (for free) and boost your sales as open source is always better then lack of support :-)
